CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Changed
 
+- 2025-08-02: remove _get_fewshot_target_text from GSM8KReasoning to make it consistent with other benchmarks that inherit from MATHReasoning and have no few-shot capability such as AIME2024 or MATH500
```

Comment:
nit-pick: I'm not sure removing a private method that raises `NotImplementedError` qualifies for something a user would like to read in a changelog :)

```suggestion
```